kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KyuubiApplicationManager.scala

```diff
@@ -115,7 +115,10 @@ object KyuubiApplicationManager {
   }
 
   private def setupFlinkYarnTag(tag: String, conf: KyuubiConf): Unit = {
-    val originalTag = conf.getOption(FlinkProcessBuilder.YARN_TAG_KEY).map(_ + ",").getOrElse("")
+    val originalTag = conf
+      .getOption(s"${FlinkProcessBuilder.FLINK}.${FlinkProcessBuilder.YARN_TAG_KEY}")
```

Comment:
even they are same string, it's for different purpose ...
```suggestion
      .getOption(s"flink.${FlinkProcessBuilder.YARN_TAG_KEY}")
```

Reply:
makes sense, let’s add a `FLINK_CONF_PREFIX`constant.